gen/com/goide/psi/GoImportSpec.java

```diff
@@ -51,4 +51,6 @@
 
   boolean isCImport();
 
+  boolean isSDKImport();
```

Comment:
We don't use abbreviations in tests

<img width="796" alt="screenshot 2016-07-17 20 10 35" src="https://cloud.githubusercontent.com/assets/140920/16901974/8d98fa3c-4c5a-11e6-8dc7-5010acc353d1.png">